### PR TITLE
Use MoleculeSearchBox in Booking portal.

### PR DESCRIPTION
**Pull request by arrogant-ant**

### CODE REVIEW

File: src/views/PageSearchPortal.vue [hunk 1]
```diff
@@ -85,28 +80,26 @@ export default {
...
             this.isLoading = false;
         },
-        async getInterestedVO() {
+        async getInterestedVO(latlng) {
             this.isLoading = true;
-            const location = this.SOLatLngInput.trim().split(',');
+            const location = latlng.trim().split(',');
             const lat = location[0].trim();
             const lng = location[1].trim();
             const parkingRequestList = await mayaClient.get(
```
Comment (NextIshwar) [inline src/views/PageSearchPortal.vue:105]: Please wrap inside try-catch block

File: src/views/PageSearchPortal.vue [hunk 2]
```diff
@@ -85,28 +80,26 @@ export default {
...
             this.isLoading = true;
             let parkingRequestURL = '/internal/parking-requests';
-            if (this.VOMobile != '') {
-                parkingRequestURL =
-                    parkingRequestURL +
-                    `?mobile=${this.VOMobile.replace(/\s+/g, '')}`;
+            if (voMobile != '') {
+                parkingRequestURL = parkingRequestURL + `?mobile=${voMobile.replace(/\s+/g, '')}`;
             }
             this.parkingRequests = await mayaClient.get(parkingRequestURL);
```
Comment (NextIshwar) [inline src/views/PageSearchPortal.vue:97]: Please wrap the API call inside try-catch

File: src/views/PageSearchPortal.vue (hunk 1 above)
Comment (arrogant-ant) [reply src/views/PageSearchPortal.vue:105]: mayaClient is handling these
https://github.com/arrogant-ant/parkspot/blob/master/src/services/api.js#L102

File: src/views/PageSearchPortal.vue (hunk 2 above)
Comment (arrogant-ant) [reply src/views/PageSearchPortal.vue:97]: mayaClient is handling these
https://github.com/arrogant-ant/parkspot/blob/master/src/services/api.js#L102

**Review by NextIshwar — APPROVED** [review]

**Pull request merged**